src/main/java/org/kiwiproject/jersey/client/RegistryAwareClient.java

```diff
@@ -28,9 +34,32 @@
     @Getter(AccessLevel.PACKAGE)
     private final RegistryClient registryClient;
 
+    /**
+     * Creates a new {@link RegistryAwareClient} with the given {@link Client} and {@link RegistryClient}. This method
```

Comment:
I don't think we need to say anything about headers for this constructor do we?

Reply:
If you want to keep it, change "method" to "constructor" since, well, it's a constructor...

Reply:
Removed.